bots/task/github.py

```diff
@@ -73,9 +64,10 @@
 TEAM_CONTRIBUTORS = "Contributors"
 
 def known_context(context):
-    for prefix in OUR_CONTEXTS:
-        if context.startswith(prefix):
-            return True
+    for project in testmap.projects():
+        for branch_tests in testmap.tests_for_project(project).values():
+            if context in branch_tests:
+                return True
```

Comment:
One problem with this - this will mark `fedora-30/verify` as unknown context although it is equivalent to `fedora-30` which would be a known context. Is this a problem and should this just split context at `\` and then split also all contexts from the map and check it exists?

Reply:
It's a bit unfortunate, but I don't think it will be a real-life problem. In principle we have this problem today already with non-cockpit problems. I think it's fine to start with the convention that the "verify" scenario isn't spelled out.